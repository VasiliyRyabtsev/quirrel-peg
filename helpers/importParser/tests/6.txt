from "file.nut" as file2 import *
