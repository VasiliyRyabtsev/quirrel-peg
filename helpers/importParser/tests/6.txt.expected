file.nut(file2): id=, path=*;